PowerShellTools/PowerShellToolsPackage.cs

```diff
@@ -130,6 +134,14 @@ public override Type GetLibraryManagerType()
             return null;
         }
 
+        public static bool UseOutProc

```

Comment:
For simplicity's sake and test costs, is there a reason we don't want do always go out of proc?